src/Providers/LumenServiceProvider.php

```diff
@@ -32,7 +32,7 @@ public function boot()
         $this->mergeConfigFrom($path, 'jwt');
 
         $this->app->routeMiddleware([
-            'jwt.auth', Authenticate::class,
+            'jwt.auth' => Authenticate::class,
```

Comment:
Otherwise this seems to yield

![screen shot 2016-03-18 at 10 01 33 am](https://cloud.githubusercontent.com/assets/3766240/13881930/10bcee60-ecf1-11e5-8889-7a2977cd32ee.png)